zebra-chain/src/sapling/shielded_data.rs

```diff
@@ -3,10 +3,29 @@ use futures::future::Either;
 use crate::{
     amount::Amount,
     primitives::redjubjub::{Binding, Signature},
-    sapling::{Nullifier, Output, Spend, ValueCommitment},
+    sapling::{tree, Nullifier, Output, Spend, ValueCommitment},
     serialization::serde_helpers,
 };
 
```

Comment:
We should probably add some doc comments to explain in general the data structure we are defining below.

Reply:
We should be able to make doc comments out of the related parts of the RFC.